src/Costumes.h

```diff
@@ -39,6 +39,7 @@ class Costumes : public PlayerScript, WorldScript
 
     // WorldScript
     void OnUpdate(uint32 diff) override;
+    void OnUpdate(Player *player, uint32 p_time) override;
```

Comment:
Could you move that a few lines up? They're sorted by script type, so it should be next to the other `PlayerScript` functions